packages/app-store/cal-ai/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://json.schemastore.org/package.json",
+  "private": true,
+  "name": "@calcom/cal-ai",
+  "version": "0.0.0",
+  "main": "./index.ts",
+  "dependencies": {
+    "@calcom/lib": "*"
+  },
+  "devDependencies": {
+    "@calcom/types": "*"
+  },
+  "description": "Cal AI is a scheduling assistant powered by GPT. Email ai@cal.com to chat with your calendar or book, edit and cancel meetings"
```

Comment:
Email does not match config.json description

Reply:
[Opted for hi@cal.ai](https://github.com/calcom/cal.com/pull/10992/commits/7eebfa1e8e12ec61b20f6fb59c7b03bf45310a8f) Thanks for pointing this out! 🙌